cdap-ui/app/cdap/components/AbstractWidget/FormInputs/SelectDropdown/index.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import TextField from '@material-ui/core/TextField';
+import MenuItem from '@material-ui/core/MenuItem';
+import ThemeWrapper from 'components/ThemeWrapper';
+import withStyles, { WithStyles } from '@material-ui/core/styles/withStyles';
+
+const styles = (theme) => {
+  return {
+    input: {
+      padding: 10,
+    },
+    root: {
+      '&$focused $notchedOutline': {
+        border: `1px solid ${theme.palette.blue[100]}`,
+      },
+    },
+    focused: {},
+    notchedOutline: {},
+  };
+};
+
+interface ISelectProps extends WithStyles<typeof styles> {
+  value: string;
+  onChange: (value: string) => void;
+  options: string[];
+  defaultVal?: string;
+}
+
+function SelectDropdown({ value, onChange, options, defaultVal, classes }: ISelectProps) {
```

Comment:
what is the difference between this and the CustomSelect?

Reply:
This is not needed. Have deleted it.